x-pack/plugins/apm/public/context/time_range_metadata/time_range_metadata_context.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { createContext } from 'react';
+import { TimeRangeMetadata } from '../../../common/time_range_metadata';
+import { useApmParams } from '../../hooks/use_apm_params';
+import { useApmRoutePath } from '../../hooks/use_apm_route_path';
+import { FetcherResult, useFetcher } from '../../hooks/use_fetcher';
+import { useTimeRange } from '../../hooks/use_time_range';
+
+export const TimeRangeMetadataContext = createContext<
```

Comment:
What information do you expect this context to hold? Having just `isUsingServiceDestinationMetrics` seems that we could rename it to something closer to that?

Reply:
See https://github.com/elastic/kibana/pull/133866#discussion_r894566861